recipes/scikit-build-core/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "scikit-build-core" %}
+{% set version = "0.1.0b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
+  sha256: 3d6bcc8ec84972a928962dd8894b82e24390f461a26fede87b4678dd1e98a8d6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - hatch-vcs
+    - hatchling
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - cmake
+    - distlib
+    - exceptiongroup
+    - importlib-resources
+    - make
+    - ninja
+    - packaging
+    - pathspec
+    - pyproject-metadata
+    - tomli
+    - typing-extensions >=3.7
+
+test:
+  imports:
+    - scikit_build_core
+  commands:
+    - pip check
+    - pytest -vv -m "not isolated" -k "not sdist_time_hash_set_epoch and not sdist_hash"
+  source_files:
+    - tests/
+    - pyproject.toml
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Are you happy just testing this on Linux? Otherwise, you can add multiple noarch platforms.

Reply:
I'd like to do that, I haven't seen that yet.

Reply:
Just add a conda-forge.yml here with the following keys:
noarch_platforms:
  - linux_64
  - win_64
  - osx_64

Thinking of it, this does require, though, that only one package is uploaded. Let me ask in the chat and maybe hold off for now.

Ref: https://conda-forge.org/docs/maintainer/conda_forge_yml.html#noarch-platforms

Reply:
OK seems to be more difficult as then three packages would be uploaded, which needs to be restricted again.

Reply:
That’s fine. The best test will be preparing awkward-cpp on all platforms. I’ll be releasing an rc in the next hour or two, I can update this (some recent testing improvements should help here).